netIDS/TimeIntervalsIDS.py

```diff
@@ -0,0 +1,168 @@
+
+#!/usr/bin/python3
+
+# Copyright (c) 2018 SecurityMatters
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors: Alexios Lekidis (alexios.lekidis@secmatters.com)
+#
+
+import sys, getopt
+from optparse import OptionParser
+import time
+from pathlib import Path
+import sqlite3
+from rawSniffer import sniff
+import sys
+from importlib import reload
+from bitstring import BitStream
```

Comment:
I think `bitstring` is not included with the python standard library.

Reply:
Yes you are right about this as well. It can be replaced by binascii which is a built-in library (https://docs.python.org/2/library/binascii.html). I will update the file now.